src/main/scala/com/itechart/ny_accidents/service/metric/PopulationMetricService.scala

```diff
@@ -18,8 +19,11 @@ class PopulationMetricService @Inject()(populationStorage: PopulationStorage) {
       populationMap.get(districtHash) match {
         case Some(value) =>
           val density = PopulationService.calculateDensity(value)
-          val ration = accidentsNumber / density
-          Some(district.districtName, ration, density, accidentsNumber)
+          val ratio: Double = accidentsNumber / density
+          NumberUtils.validateDouble(ratio) match {
```

Comment:
Maybe it'd be much easy to check if divisor(i.e. _density)_ is zero and return None in such case?

Reply:
Yes, it looks better.